### ISSUE TITLE
Installing not as root

### ISSUE THREAD

**Issue by adamjgrant** [buckket/twtxt#2]
> Tip: Instead of installing the package globally (as root), you may want to install this package locally by passing --user to pip, but you then have to make sure that you have included ~/.local/bin/ in your $PATH. Using pyvenv and running twtxt from within a virtualenv is of course also an option!

Could you elaborate on this one for python noobs?

When I try `pip3 install twtxt --user=Adam` I get "--user option does not take a value"

**Comment by syberghost** [comment]
--user with no options, running as Adam, not root. Error means exactly what it says.

**Comment by adamjgrant** [comment]
I trust the error means exactly what it says. Thank you.

Can you explain how to resolve please?
Le Fri, Feb 5, 2016 à 12:30 PM, Shawn McMahon <notifications@github.com> a
écrit :

> --user with no options, running as Adam, not root. Error means exactly
> what it says.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/buckket/twtxt/issues/2#issuecomment-180545332>.
>
-- 
Adam Grant

**Comment by Necior** [comment]
@adamjgrant: just run `pip3 install twtxt --user` as non-root user. Take a look:

```
n3t@velox:~$ pip3 install twtxt --user
Downloading/unpacking twtxt
(...)
n3t@velox:~$
```

**Comment by Regner** [comment]
@adamjgrant You ran the command as ```pip3 install twtxt --user=Adam``` and the error said ```--user option does not take a value```. When you write ```=Adam``` you're passing Adam as a value to the user paramater. So as @syberghost pointed out the message is telling you exactly what to fix, don't pass an argument. If you log into the Adam user and just run ```pip3 install twtxt --user``` you should be fine.

**Comment by adamjgrant** [comment]
Thank you
Le Sat, Feb 6, 2016 à 11:42 AM, buckket <notifications@github.com> a écrit :

> Closed #2 <https://github.com/buckket/twtxt/issues/2>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/buckket/twtxt/issues/2#event-541852865>.
>
-- 
Adam Grant
